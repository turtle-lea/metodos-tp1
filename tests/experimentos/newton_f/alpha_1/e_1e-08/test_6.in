1 20 1e-08 6
#