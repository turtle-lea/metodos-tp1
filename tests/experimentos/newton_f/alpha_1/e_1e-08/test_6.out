1
3 8
1.66667 1.77778
